apps/ledger-live-mobile/package.json

```diff
@@ -154,6 +154,7 @@
     "hoist-non-react-statics": "3.3.2",
     "i18next": "20.6.1",
     "invariant": "2.2.4",
+    "jotai": "^2.7.0",
```

Comment:
Small detail but we could update jotai to the latest version, it includes some fixes